task1.sh

```diff
@@ -0,0 +1,50 @@
+<<"COMMENT"
+Write a script to search a file in a given folder.
+~$ ./task1.sh <search_file> <from_folder>
+For example:
+./task1.sh file1 folder1/
+Should output searching every single occurrence of file1 by using depth first search.
+COMMENT
+
+#!/bin/bash
+
+#The first thing we will do is check whether the user passed the file and folder name in the args
+#The -z variable tells us if argument is empty string or not 
+if [ -z "$1" ] || [ -z "$2" ]; then
+    echo "Correct Format is following: $0 <file> <folder>"
+    echo "Kindly re-run the script with the correct format"   
+    exit 1
+fi
+
+#we can declare as much arguments as we like here
```

Comment:
why we need this comment?

Reply:
I added this comment to tell that we can add more argument by simply doing $3 and so on, if it is unnecessary I will remove it